packages/core/package.json

```diff
@@ -29,6 +29,9 @@
     "dev": "vite build --watch",
     "test": "vitest"
   },
+  "dependencies": {
+    "@module-federation/runtime": "^0.1.2"
```

Comment:
verify it works properly in node.js env (plus no `eval`)

Reply:
@avimak

This is how federation loads code in node.js environment (using vm):
https://github.com/module-federation/universe/blob/920d9869836dfe477e0139673ca3714248fb8827/packages/sdk/src/node.ts#L58

This is how federation loads script in browser environment:
https://github.com/module-federation/universe/blob/920d9869836dfe477e0139673ca3714248fb8827/packages/sdk/src/dom.ts#L182

This is how federation loads es module:
https://github.com/module-federation/universe/blob/920d9869836dfe477e0139673ca3714248fb8827/packages/runtime/src/utils/load.ts#L22

Since native modules and features of node are used to load external code, I find it safe enough.
What do you think? Is there a way to verify with Chrome store that those methods are fine?